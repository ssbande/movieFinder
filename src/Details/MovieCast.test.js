import React from 'react';
import { shallow } from 'enzyme';

import MovieCast from './MovieCast';

const setup = () => ({
  props: {
    cast: [{
      id: 1001,
      profile_path: 'profile_path',
      name: 'name1',
      character: 'character1'
    }],
    handleCardClick: jest.fn(),
  }
})

describe('Movie Cast', () => {
  it('should render correctly', () => {
    const { props } = setup();
    const wrapper = shallow(<MovieCast {...props} />);
    expect(wrapper).toMatchSnapshot();
  })
})