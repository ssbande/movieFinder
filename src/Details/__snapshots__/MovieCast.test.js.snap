// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Cast should render correctly 1`] = `ShallowWrapper {}`;
